libc/src/__support/UInt.h

```diff
@@ -244,18 +244,30 @@ LIBC_INLINE constexpr bool is_negative(cpp::array<word, N> &array) {
 enum Direction { LEFT, RIGHT };
 
 // A bitwise shift on an array of elements.
+// TODO: Make the result UB when 'offset' is greater or equal to the number of
```

Comment:
Or similar to other ones, we can have both safe and unsafe shifts.

Reply:
I'd like to do this as a separate PR because some tests and code seem to depend on this behavior.